src/plugins/percentage-fontsize.js

```diff
@@ -0,0 +1,134 @@
+/**
+ * SCEditor Inline-Code Plugin for BBCode format
+ * http://www.sceditor.com/
+ *
+ * Copyright (C) 2011-2013, Sam Clarke (samclarke.com)
+ *
+ * SCEditor is licensed under the MIT license:
+ *	http://www.opensource.org/licenses/mit-license.php
+ *
+ * @fileoverview SCEditor percentage fontsize plugin for BBCode format
+ * This plugin allow usage of the percent indication in font size bbcode tag
+ * in a format used in phpBB system:
+ * [size=50]text[/font]
+ * means the text will be shown with font 50% of the current size
+ * [size=200]text[/font]
+ * means the text will be shown with font 200% of the current size
+ * @author Alex Betis
+ */
+
+(function (sceditor) {
+	'use strict';
+
+	var dom = sceditor.dom;
+
+	sceditor.plugins['percentage-fontsize'] = function () {
+		var base = this;
+
+		/**
+		 * Default sizes
+		 * @type {Object}
+		 * @private
+		 */
+		var sizes = [50, 75, 100, 150, 200];
+
+		/**
+		 * Private functions
+		 * @private
+		 */
+		var	commandHandler;
+
+		base.init = function () {
+			var	opts  = this.opts;
+
+			// Enable for BBCode only
+			if (opts.format && opts.format !== 'bbcode') {
+				return;
+			}
+
+			// The plugin will override current fontsize implementation
+			sceditor.command.set('size', {
+				exec: commandHandler,
+				txtExec: commandHandler,
+				tooltip: 'Font Size'
```

Comment:
Use `_` function to make it translatable or allow it to be translable somehow.

Reply:
Please explain what do you mean here.
`_` for what function?
What does "translatable" means?

Reply:
It's a function named `_`. See: https://www.sceditor.com/api/sceditor/underscore/
"translatable" <=> To be able to have a different text on the node based on the current language

Reply:
Sorry, but I don't understand what function do you mean that have to be with underscore or translatable.
I'm basing my code the same way as is appears in the format plugin that Sam wrote and as defaultCommands.js code looks like...

Reply:
Ah, OK. The code sends tooltips through the translation function when displaying:
https://github.com/samclarke/SCEditor/blob/master/src/lib/SCEditor.js#L712
I had forgotten about that.